frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/storage/UploadImageModel.java

```diff
@@ -169,22 +169,13 @@ public void initialize() {
                     getDataCenter().setIsChangeable(limitToStorageDomainId == null);
```

Comment:
I would suggest some code refactoring here, since you touch it anyway:
1) Define a new local variable `boolean isChangeable = limitToStorageDomainId == null`. And use it 2-3 times.
2) Rename `initUnchangeableDcAndDomain` to `initUnchangeableDcAndSd`. Otherwise there is some inconsistency - you use a shortcut `Dc` for Data Center, while a full (but 1-word) `Domain` for Storage Domain. BTW, is Data Center relevant at all for naming the method, or only Storage Domain?
3) Call the above method only if `!isChangeable`. This will simplify the new method - no need to test for `limitToStorageDomainId` inside it.
4) Inside the above method simply rename `limitToStorageDomainId` into `storageDomainId`.